components/chart/types/column.md

```diff
@@ -59,6 +59,17 @@ Column series
 
 ## Column Chart Specific Appearance Settings
 
+### Markers
+
+Each data item is denoted with a marker (label). You can control and customize them through the `< ChartCategoryAxisLabels />` and its children tags.
+
+* `Visible` - hide all markers by setting this parameter to `false`.
+* `Step` - renders every n-th marker, where n is the value(double number) passed to the parameter.
+* `Skip` - skips the first n markers, where n is the value(double number) passed to the parameter.
+* `Angle` - rotates the markers with the desired angle by n degrees, where n is the value passed to the parameter. It can take positive and negative numbers. To set this parameter use the `< ChartCategoryAxisLabelsRotation />` child tag.
+
+To rotate the markers use the `ChartCategoryAxisLabelsRotation` child tag and set its `Angle` parameter. It can take positive and negative numbers as value.
```

Comment:
Isn't this in the next template about configurable nested properties?

Reply:
If so, maybe move this new section after the Color section so they are grouped together